settings/base.py

```diff
@@ -3,7 +3,6 @@
 """
 
 import os
-from celery import Celery
 
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
```

Comment:
I believe you should remove TEMPLATE_DEBUG. You can include 'debug' in the TEMPLATES options.

For example,

````
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [
            os_path.join(PROJECT_PATH, 'templates'),
        ],
        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                "django.contrib.auth.context_processors.auth",
                "django.template.context_processors.debug",
                "django.template.context_processors.i18n",
                "django.template.context_processors.media",
                "django.template.context_processors.static",
                "django.template.context_processors.request",
                "django.template.context_processors.tz",
                "django.contrib.messages.context_processors.messages",
                "sekizai.context_processors.sekizai",
            ],
            'debug': DEBUG,
        },
    },
]